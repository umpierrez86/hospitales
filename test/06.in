leer_instancia
4 10
2 2 3 2
0 1 2 3 4 5 6 7 8 9
0 1 2 3 4 5 6 7 8 9
9 8 7 6 5 4 3 2 1 0
9 8 7 6 5 4 3 2 1 0
0 1 2 3
3 1 0 2
1 3 2 0
1 0 2 3
2 0 1 3
0 1 2 3
3 1 0 2
1 3 2 0
1 0 2 3 
2 0 1 3
imprimir_instancia
asignar_residencias
imprimir_asignacion
Fin