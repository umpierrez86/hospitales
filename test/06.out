1>2>Cantidad hospitales: 4
Cupos hospitales: 2 2 3 2 
Cantidad estudiantes: 10
Preferencias hospitales
Preferencias hospital h0: 0 1 2 3 4 5 6 7 8 9 
Preferencias hospital h1: 0 1 2 3 4 5 6 7 8 9 
Preferencias hospital h2: 9 8 7 6 5 4 3 2 1 0 
Preferencias hospital h3: 9 8 7 6 5 4 3 2 1 0 
Preferencias estudiantes
Preferencias estudiante e0: 0 1 2 3 
Preferencias estudiante e1: 3 1 0 2 
Preferencias estudiante e2: 1 3 2 0 
Preferencias estudiante e3: 1 0 2 3 
Preferencias estudiante e4: 2 0 1 3 
Preferencias estudiante e5: 0 1 2 3 
Preferencias estudiante e6: 3 1 0 2 
Preferencias estudiante e7: 1 3 2 0 
Preferencias estudiante e8: 1 0 2 3 
Preferencias estudiante e9: 2 0 1 3 
3>4>(0, 0) (1, 1) (2, 1) (3, 0) (4, libre) (5, 2) (6, 3) (7, 3) (8, 2) (9, 2) 
5>Fin.
